### ISSUE TITLE
not view war file in deployement file location

### ISSUE THREAD

**Issue by ruwan1235** [bitnami/charts#6880]
I used bitnami/wildfly official helm chart (version 11.0.1)

i add added war file using wildfly dashboard.show bellow screenshot.
![Screenshot from 2021-07-07 18-28-42](https://user-images.githubusercontent.com/86238699/124763152-46e9bb00-df51-11eb-867a-2127339a6489.png)

but not show that war file in docker image (path - /opt/bitnami/wildfly/standalone/deployments/)

After i will add war file to docker image " /opt/bitnami/wildfly/standalone/deployments" location.

we can view that war file using admition console. 

But we can't find war file location, added using admition console.

**Comment by alemorcuq** [comment]
Check in `/opt/bitnami/wildfly/standalone/tmp`

**Comment by ruwan1235** [comment]
> Check in `/opt/bitnami/wildfly/standalone/tmp`
@alemorcuq i checked that location all deployement there. But after mount that location and we can't add war file using Admition console.

And where the location alredy added  datasource configuration file

**Comment by alemorcuq** [comment]
I'm sorry, @ruwan1235, I didn't understand your question. Could you rephrase it, please?
